_data/tool_and_resource_list.yml

```diff
@@ -1,3 +1,16 @@
+# List of tools and resources
```

Comment:
```suggestion
# List of tools and resources
# Conforms to https://w3id.org/everse/rs/0.0.1
```
I will publish the metadata schema in https://w3id.org/everse/rs/0.0.1 (soon)

Reply:
ok - will wait for it to be published before I merge

Reply:
Please go ahead an publish, I will make it available soon